mobile/config.xml

```diff
@@ -29,9 +29,6 @@
         <preference name="StatusBarStyle" value="default" />
         <allow-intent href="itms:*" />
         <allow-intent href="itms-apps:*" />
-        <config-file mode="add" parent="ITSAppUsesNonExemptEncryption" target="*-Info.plist">
-            <false />
-        </config-file>
```

Comment:
When you publish a new version on Apple Store, they are a question:  It's app uses encryption?

Normally this code respond to false but it's never worked... so I decide to find another solution. With cordova-plugin-ios-no-export-compliance it's now ok 👍